products/sle12/profiles/stig.profile

```diff
@@ -260,6 +260,7 @@ selections:
     - sysctl_net_ipv4_icmp_echo_ignore_broadcasts
     - sysctl_net_ipv4_ip_forward
     - sysctl_net_ipv4_tcp_syncookies
+    - sysctl_net_ipv6_conf_all_accept_redirects
```

Comment:
I think this line is superfluous in this PR. The changes are to forwarding and source_route... And forwarding isn't in this STIG profile and should probably be added. 

So I'd change this line to:

```yaml
 - sysctl_net_ipv6_conf_default_forwarding
```

Or drop it altogether. Note that it is breaking the build currently as-is. :-)

Reply:
Thanks for the PR though! :)

Reply:
Ah it was probably a bad copy-paste with #7165.

Reply:
Hey! Yes you are right, looks like I just made a little copy+paste mistake. :) I will fix it shortly, thanks for your help!